src/main/java/org/wildfly/extension/elytron/IdentityResourceDefinition.java

```diff
@@ -740,21 +720,18 @@ public void validateParameter(String parameterName, ModelNode value) throws Oper
     private static List<Object> getCredentials(final ModifiableRealmIdentity realmIdentity) throws RealmUnavailableException {
         List<Object> credentials = new ArrayList<>();
 
-        addPassword(realmIdentity, BCryptPassword.class, credentials);
-        addPassword(realmIdentity, ClearPassword.class, credentials);
-        addPassword(realmIdentity, SimpleDigestPassword.class, credentials);
-        addPassword(realmIdentity, SaltedSimpleDigestPassword.class, credentials);
-        addPassword(realmIdentity, DigestPassword.class, credentials);
+        //TODO: accordingly with David, we should now obtain the credential names instead of the actual credential based on a type. However, this is not being considered in ELY-282 yet. Once this capability (obtain credential names) is implemented, need to change this method.
```

Comment:
Here we support describing the credentials associated with a given identity during a read identity operation. Until now, it was possible to obtain the credentials based on a type, but with ELY-282 we can't do that anymore.

After a quick discussion with @dmlloyd, we would need a method that can obtain all the credential names from a realm identity. Once this is implemented, we can change that part of the code and make it display the credential names associated with a given identity.